lib/sles4sap/ipaddr2.pm

```diff
@@ -963,7 +993,8 @@ sub ipaddr2_cloudinit_sanity {
             'sudo systemctl status cloud-final.service') {
             ipaddr2_ssh_internal(id => $id,
                 cmd => $_,
-                bastion_ip => $args{bastion_ip});
+                bastion_ip => $args{bastion_ip},
+                tiemout => 180);
```

Comment:
if you know about a specific command that is slower and needs this longer timeout here, please add a comment in the code to document which one is problematic.